android/app-newm/src/main/java/io/newm/screens/Screen.kt

```diff
@@ -4,34 +4,14 @@ import kotlinx.parcelize.Parcelize
 import com.slack.circuit.runtime.screen.Screen as CircuitScreen
 
 @Parcelize
-sealed class Screen(val route: String) : CircuitScreen {
-    //High Navigation Roots
-    data object HomeRoot : Screen("home-root")
-    data object HomeLanding : Screen("home-landing")
+sealed class Screen : CircuitScreen {
```

Comment:
This looks much better! 

At some point once the app navigation grows some more might be worth having a core module where we define all screens we can navigate to.